pyproject.toml

```diff
@@ -1,3 +1,15 @@
+[project]
+name = "cramjam"
+license = {file = "LICENSE"}
+keywords = ["compression", "decompression", "snappy", "zstd", "bz2", "gzip", "lz4", "brotli", "deflate"]
+requires-python = ">=3.7"
+readme = "README.md"
```

Comment:
No need to specify it again, maturin will use the Cargo.toml readme field automatically.